src/jwt/token_manager.rs

```diff
@@ -49,11 +51,36 @@ impl TokenManager {
 }
 
 fn get_keys() -> Result<(RS384KeyPair, RS384PublicKey), Box<dyn Error>> {
-    let private_key_content = fs::read_to_string("keys/private.pem")?;
-    let public_key_content = fs::read_to_string("keys/public.pem")?;
+    let private_key_content = fs::read_to_string(var("KEY_DIR").unwrap() + "/private.pem")?;
```

Comment:
Der burde nok være lidt mere error handling mange steder. Generalt plejer man at sige, at det er dårlig practice bare og bruge unwrap. Det er nok mere sikkert og bruge andre metoder, som at matche det, eller idemindste bruge en anden metode end unwrap som ikke bare vil panicce hvis den ikke finder noget

Reply:
Hvad jeg har tænkt er, at funktioner som disse, der kun bliver brugt i setup af kritiske komponenter (token manager, in this case), godt må panicce, da applikationen ikke rigtig fungerer hvis ikke de kører korrekt.